server/src/test/java/org/apache/calcite/test/ServerTest.java

```diff
@@ -242,6 +244,26 @@ static Connection connect() throws SQLException {
     }
   }
 
+  @Test public void testInsertCreateNewCompositeUdt() throws Exception {
+    try (Connection c = connect();
+        Statement s = c.createStatement()) {
+      boolean b = s.execute("create type mytype as (i int, j int)");
+      assertThat(b, is(false));
+      b = s.execute("create table w (i int not null, j mytype)");
+      assertThat(b, is(false));
```

Comment:
Please refrain from using `assertThat(actual, is(false));`.
See https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md#test

As a general rule, a test failure (class name + method name + failure message) should look like a good bug report.

If this `assertThat(b, is(false));` fails, the error would be  like

`ServerTest.testInsertCreateNewCompositeUdt  failed with AssertionError:  expected: false got: true` which is truly obscure and sad.

Reply:
Done :)

Reply:
Change `assertThat(b, is(false))` to `assertFalse(b)`.

Reply:
Done